docs/demand/historical/eastern.rst

```diff
@@ -0,0 +1,26 @@
+Eastern
+#######
+The construction of the Eastern Interconnection demand profile is slightly different
+from the Western counterpart. We still download historical demand data from EIA
+and perform the 3-step data processing/cleaning as for the Western Interconnection
+for all Balancing Authorities (BAs) but MISO (Midcontinent Independent System Operator)
+and SPP (Southwest Power Pool).
+
+MISO and SPP operate across multiple U.S. states (e.g. 15 for MISO) and cover several
+load zones in our grid model. Instead of considering a single hourly profile shape for
+each entity and having load zones sharing the exact same profile in different part of
+the country, we further split MISO and SPP into subareas. The hourly subarea demand
+profiles for MISO and SPP can be downloaded on their respective website (check `SPP
+hourly load`_ and `MISO hourly load`_). The geographical definitions of these subareas
+are obtained directly from contact at the BAs via either customer service or internal
+request management system (see the `RMS`_ for SPP). Given the geographical definitions,
+i.e., list of counties for each subarea, each bus is mapped to the subarea it belongs
```

Comment:
To be specific, maybe we could add "each bus within MISO and SPP is mapped to the subarea it belongs to."

Reply:
Done